### ISSUE TITLE
Can't use full webdav URL

### ISSUE THREAD

**Issue by llaumgui** [JuanJakobo/Pocketbook-Nextcloud-Client#40]
Hi,

I would like to use a full webdav URL to prevent a full sync of my nextcloud files structure. Like that:
https://domainname/remote.php/dav/files/UUID/Folder1/Folder2/ebooks

But, currently, if I use this URL, I have a 405 error.

Best regards.

**Comment by JuanJakobo** [comment]
Hi,
The structure will never be synced automatically. Only the metadata will be saved to the database, but the files will only be downloaded if you select them. 
To reach your goal you could select "Exclude and hide items" in the main menu and in this menu set a startfolder (in your example /Folder1/Folder2/). After doing so the app will open in that directory. Here you can do a long click on the folder ebooks and select sync, this will download the content of that folder. Once it gets out of sync, you can do the long press again and it will adapt the changes. The application has currently only read access to your file structure.
Bests

**Comment by llaumgui** [comment]
Yes, but on my PB file system, I have the full structure.

**Comment by JuanJakobo** [comment]
Yes, that is an expected behavior.

**Comment by llaumgui** [comment]
Is there no way to sync a part of a tree and not the full tree ? Currently, I sync only my wanted tree, but I have the full tree structure, and it's not pretty to display.

**Comment by JuanJakobo** [comment]
Thats true, I will have a look into it, once I have a little bit of time left for it.
